spec/unit/action/deploy/environment_spec.rb

```diff
@@ -147,11 +147,11 @@
           and_return(loader).at_least(:once)
       end
 
-      it "assume unchanged flag causes the module definitons to be preloaded by the loader" do
+      it "incremental unchanged flag causes the module definitons to be preloaded by the loader" do
```

Comment:
```suggestion
      it "incremental flag causes the module definitons to be preloaded by the loader" do
```

Reply:
smh, I guess it really shows that my search term was just "assume"